lidar_point_pillars/CMakeLists.txt

```diff
@@ -55,22 +55,37 @@ else()
   set(CUDNN_AVAIL OFF)
 endif()
 
-if(TRT_AVAIL AND CUDA_AVAIL AND CUDNN_AVAIL)
+# Check TVM utility, runtime libraries and TVM flag
+option(TVM_AVAIL "TVM implementation available" OFF)
+find_library(TVM_LIBRARY NAMES tvm_runtime)
+
+if(CUDA_AVAIL AND ((TRT_AVAIL AND CUDNN_AVAIL) OR (TVM_AVAIL AND TVM_LIBRARY)))
   find_package(catkin REQUIRED COMPONENTS
     roscpp
     roslib
     pcl_ros
     autoware_msgs
   )
-  set(CMAKE_CXX_STANDARD 11)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
   catkin_package(
     CATKIN_DEPENDS
       roslib
       pcl_ros
       autoware_msgs
   )
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+
+  # Look for tvm_utility if this build is using TVM model
+  if(TVM_AVAIL)
+    find_package(catkin REQUIRED COMPONENTS
+      tvm_utility
+    )
```

Comment:
Since `tvm_utility` is in `package.xml`, you can just make it REQUIRED.

Reply:
As in `find_package(tvm_utility REQUIRED)`? That makes us lose the reference to its header files, and we would need to `find_path` it. I'm not sure that's better.

Reply:
Sorry, no, I meant without the `TVM_AVAIL`. However, now that I think about it, there's no particular reason to link against it if we aren't using it so this is fine.